### PR TITLE
Move assets

### LINKED ISSUE #787: Remove assets folder
The `assets` folder only contains 3 subfolders. I prefer to remove the assets folder and put `languages`, `styles` and `scripts` in the theme root folder.

How do you guys feel about this now?

### PR DESCRIPTION
**Pull request by sboerrigter**
Fixes #787

### CODE REVIEW

File: lib/classes/Admin/Editor.php [hunk 1]
```diff
@@ -12,7 +12,7 @@ public function __construct()
 
     public function loadStyles()
     {
-        add_editor_style('assets/styles/output/editor.css');
+        add_editor_style('styles/output/editor.css');
```
Comment (haroldangenent) [inline lib/classes/Admin/Editor.php]: Editor styles seemed to stop working altogether. This seems to be a regression bug after #804.

File: lib/classes/Assets.php [hunk 2]
```diff
@@ -10,8 +10,8 @@ public function __construct()
 
     public function frontend()
     {
-        $assetsPath = get_stylesheet_directory() . '/assets';
-        $assetsUri = get_stylesheet_directory_uri() . '/assets';
+        $assetsPath = get_stylesheet_directory();
+        $assetsUri = get_stylesheet_directory_uri();
```
Comment (haroldangenent) [inline lib/classes/Assets.php]: These temp (query) variables don't seem necessary anymore and only makes this code less readable. Could you remove them?

File: lib/classes/Admin/Editor.php (hunk 1 above)
Comment (sboerrigter) [reply lib/classes/Admin/Editor.php]: Oops. Fixed it now.

File: lib/classes/Assets.php (hunk 2 above)
Comment (sboerrigter) [reply lib/classes/Assets.php]: Hooray!

**Comment by sboerrigter** [comment]
Oops... 😳

**Pull request merged**